### ISSUE TITLE
DataGrid: Sorting Status is not identified by screen reader

### ISSUE THREAD

**Issue by kalpeshhpatel** [vmware/clarity#4093]
## Describe the bug

In Datagrid, when the column is sorted either ascending/descending, nothing is read by the screen reader. 

1. Go to 'https://v2.clarity.design/datagrid/full
2. Click on the 'Name' column to sort.
3. Use the NVDA screen reader.
4. Column sorts in proper order.
5. However, the screen reader reads nothing about the change in this sorting status.

## Expected behavior
The screen reader should be able to recognize the status of the sortable column if it's sorted ascending or descending.  

## Versions

This issue is reproducible with latest version of Clarity.

<img width="1331" alt="Screen Shot 2019-12-06 at 2 26 47 PM" src="https://user-images.githubusercontent.com/5223493/70310792-7d6fc900-1836-11ea-9b93-fe8acee8417c.png">

**Comment by Jinnie** [comment]
Tested with versions from v2 up to v4. Sort order is reported (see screenshot):

<img width="617" alt="Screen Shot 2020-10-21 at 11 05 47 AM" src="https://user-images.githubusercontent.com/1798828/96691490-9f5e3580-138d-11eb-917f-7495d7779ada.png">

**Comment by Jinnie** [comment]
This is per W3C specification and follows the examples here:
https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html

**Comment by Jinnie** [comment]
It was discovered that the problem here is valid, as in NVDA the sorting is not announced. I will not reopen this issue though, because we handle our accessibility issues in internal system. I will update the labels, though.
It's also not clear if this can be considered NVDA problem, as our code confirms to the specs.

**Comment by Jinnie** [comment]
A possibly related NVDA bug:
https://github.com/nvaccess/nvda/issues/10890
